Fix expected source help text for statistics graph unit mismatch (#44424)
home-assistant/home-assistant.github.io: source/more-info/statistics.markdown

Check that it hasn't been excluded in the [Recorder](https://www.home-assistant.io/integrations/recorder/) configuration.

> Make sure the entity uses the same unit as the already selected sensors for that statistics graph. The statistics card uses one Y axis, so mixing units (for example, °C and % or kW and W) is not supported in the same graph.

Reviewer: ```suggestion
Make sure the entity uses the same unit as the already selected sensors for that statistics graph. Because the graph uses a single Y axis, you cannot mix units (for example, °C and %, or kW and W) in the same graph.
```
